### PR TITLE
Example JSON-LD

### PR DESCRIPTION
**Pull request by tlodderstedt**
This is an example of a JSON-LD based credential conveyed in a SD-JWT. SD-JWT provides integrity, authenticity, and selective disclosure for the enclosed JSOMN-LD object. The payload (after processing the SD-JWT) is a JSON-LD object that can be fed into a JSON-LD processor. 
The concrete example is a Vaccination Certificate according to https://w3c-ccg.github.io/vaccination-vocab/.

### CODE REVIEW

File: examples/jsonld/sd_jwt_payload.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/vaccination/v1"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "VaccinationCertificate"
+  ],
+  "issuer": "https://example.com/jdoe",
```
Comment (Sakurann) [inline examples/jsonld/sd_jwt_payload.json:10]:
```suggestion
  "issuer": "https://example.com/issuer",
```

File: examples/jsonld/user_claims.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/vaccination/v1"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "VaccinationCertificate"
+  ],
+  "issuer": "https://example.com/jdoe",
```
Comment (Sakurann) [inline examples/jsonld/user_claims.json:10]:
```suggestion
  "issuer": "https://example.com/issuer",
```

File: examples/jsonld/verified_contents.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/vaccination/v1"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "VaccinationCertificate"
+  ],
+  "issuer": "https://example.com/jdoe",
```
Comment (Sakurann) [inline examples/jsonld/verified_contents.json:10]:
```suggestion
  "issuer": "https://example.com/issuer",
```

File: sd_jwt/examples/jsonld.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
+user_claims:
+  {
+    "@context":
+      [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/vaccination/v1",
+      ],
+    "type": ["VerifiableCredential", "VaccinationCertificate"],
+    "issuer": "https://example.com/jdoe",
```
Comment (Sakurann) [inline sd_jwt/examples/jsonld.yml:9]:
```suggestion
    "issuer": "https://example.com/issuer",
```

File: examples/jsonld/sd_jwt_payload.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        "8FbBeqe_wV1ES-RU28y8DDkfQv9dlGziVdd9R3jq-fQ",
+        "Cu8JNGQ2WRs6KHKngBKEaU9r6SMfnwoVVYGTPWwvxJc",
+        "YJI42JtoLHmwv2oi45QW6wmINZXTpYBxhD4eWlmEpiw"
+      ],
+      "type": "VaccineRecipient"
+    },
+    "type": "VaccinationEvent"
+  },
+  "iss": "https://example.com/issuer",
+  "iat": 1516239022,
```
Comment (Sakurann) [inline examples/jsonld/sd_jwt_payload.json:45]:
```suggestion
  "nbf": 1516239022,
```

It's not `exp` per `nbf MUST represent issuanceDate, encoded as a UNIX timestamp (NumericDate).`

File: sd_jwt/examples/jsonld.yml [hunk 6]
```diff
@@ -0,0 +1,71 @@
+user_claims:
+  {
+    "@context":
+      [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/vaccination/v1",
```
Comment (Sakurann) [inline sd_jwt/examples/jsonld.yml:6]:
```suggestion
        "https://w3id.org/vaccination/v1"
```

File: sd_jwt/examples/jsonld.yml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    "name": "COVID-19 Vaccination Certificate",
+    "description": "COVID-19 Vaccination Certificate",
+    "credentialSubject":
+      {
+        "vaccine":
+          {
+            "type": "Vaccine",
+            "atcCode": "J07BX03",
+            "medicinalProductName": "COVID-19 Vaccine Moderna",
+            "marketingAuthorizationHolder": "Moderna Biotech",
```
Comment (Sakurann) [inline sd_jwt/examples/jsonld.yml:21]:
```suggestion
            "marketingAuthorizationHolder": "Moderna Biotech"
```

File: sd_jwt/examples/jsonld.yml [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+        "countryOfVaccination": "GE",
+        "dateOfVaccination": "2021-06-23T13:40:12Z",
+        "order": "3/3",
+        "recipient":
+          {
+            "type": "VaccineRecipient",
+            "gender": "Female",
+            "birthDate": "1961-08-17",
+            "givenName": "Marion",
+            "familyName": "Mustermann",
```
Comment (Sakurann) [inline sd_jwt/examples/jsonld.yml:33]:
```suggestion
            "familyName": "Mustermann"
```

File: sd_jwt/examples/jsonld.yml [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+          },
+        "type": "VaccinationEvent",
+        "administeringCentre": "Praxis Sommergarten",
+        "batchNumber": "1626382736",
+        "healthProfessional": "883110000015376",
+      },
```
Comment (Sakurann) [inline sd_jwt/examples/jsonld.yml:39]:
```suggestion
      }
```

**Review by Sakurann — COMMENTED** [review]
[passed JSON-LD playground](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%5B%22https%3A%2F%2Fwww.w3.org%2F2018%2Fcredentials%2Fv1%22%2C%22https%3A%2F%2Fw3id.org%2Fvaccination%2Fv1%22%5D%2C%22type%22%3A%5B%22VerifiableCredential%22%2C%22VaccinationCertificate%22%5D%2C%22issuer%22%3A%22https%3A%2F%2Fexample.com%2Fjdoe%22%2C%22issuanceDate%22%3A%222023-02-09T11%3A01%3A59Z%22%2C%22expirationDate%22%3A%222028-02-08T11%3A01%3A59Z%22%2C%22name%22%3A%22COVID-19%20Vaccination%20Certificate%22%2C%22description%22%3A%22COVID-19%20Vaccination%20Certificate%22%2C%22credentialSubject%22%3A%7B%22vaccine%22%3A%7B%22type%22%3A%22Vaccine%22%2C%22atcCode%22%3A%22J07BX03%22%2C%22medicinalProductName%22%3A%22COVID-19%20Vaccine%20Moderna%22%2C%22marketingAuthorizationHolder%22%3A%22Moderna%20Biotech%22%7D%2C%22nextVaccinationDate%22%3A%222021-08-16T13%3A40%3A12Z%22%2C%22countryOfVaccination%22%3A%22GE%22%2C%22dateOfVaccination%22%3A%222021-06-23T13%3A40%3A12Z%22%2C%22order%22%3A%223%2F3%22%2C%22recipient%22%3A%7B%22type%22%3A%22VaccineRecipient%22%2C%22gender%22%3A%22Female%22%2C%22birthDate%22%3A%221961-08-17%22%2C%22givenName%22%3A%22Marion%22%2C%22familyName%22%3A%22Mustermann%22%7D%2C%22type%22%3A%22VaccinationEvent%22%2C%22administeringCentre%22%3A%22Praxis%20Sommergarten%22%2C%22batchNumber%22%3A%221626382736%22%2C%22healthProfessional%22%3A%22883110000015376%22%7D%7D)

File: examples/jsonld/sd_jwt_payload.json (hunk 5 above)
Comment (tlodderstedt) [reply examples/jsonld/sd_jwt_payload.json:45]: I think the JWT claims should be removed/be replaced by JSON-LD claims (like iss must be issuer, iat must be issuanceDate).

**Review by Sakurann — APPROVED** [review]
thank you - Looks good.

**Pull request merged**